sites/game-client/ui/main-ui.mjs

```diff
@@ -43,28 +64,90 @@ class MainUI {
    * 
    * @param target The element which wants add button, should be an Overlay or Modal.
    */
-  addToolbarButton(target)
+  addToolbarButton(target) {
+    // TODO(fanlan1210)
+  }
 
   /**
    * Remove an button from the Toolbar.
    * 
    * @param target The element which wants remove button, should be an Overlay or Modal.
    */
-  removeToolbarButton(target)
+  removeToolbarButton(target) {
+    // TODO(fanlan1210)
+  }
 
   // TODO: Stack visualization? Stack height limit?
   /**
    * Show notification in notification area.
    * @param msg The message want to be shown.
    * @param {Number} timeout The duraion of showing notification, the unit is millisecond.
    */
-  showNotification(msg, timeout);
+  showNotification(msg, timeout) {
+    // TODO(lisasasasa)
+  }
 
   // If we want to use individual overlay for meeting
   // position = 0...n
   // setTopbarOverlay(position, overlay);
   // count=0 for no topbar.
   // resizeTopbarOverlay(count);
+
+  // ========== Internal Methods ============
+  // Methods below should only be used by UI classes such as Model/Overlay.
```

Comment:
Maybe start below function with an underscore as a programmer's hint, since not everyone reads this comment.

Reply:
That was not done because JS doesn't have friend class so marking them private may cause issues with some linters. However, if you've concerns we'll go that way. Underscore added.